docs/operator.md

```diff
@@ -628,3 +628,61 @@ file contains configuration options to customize the PostgreSQL high-availabilit
 | **Example**     | <pre>postgresql:<br>  parameters:<br>    max_parallel_workers: 2<br>    max_worker_processes: 2<br>    shared_buffers: 1GB<br>    work_mem: 2MB</pre> |
 | **Description** | Custom PostgreSQL configuration options. Please note that configuration changes are automatically applied to the running instances without validation, so having an invalid config can make the cluster unavailable |
 
+## Custom extensions Section
+
+The `extensions` section in the [deploy/cr.yaml](https://github.com/percona/percona-postgresql-operator/blob/main/deploy/cr.yaml)
+file contains configuration options to [manage PostgreSQL extensions](custom-extensions.md).
+
+|                 | |
+|-----------------|-|
+| **Key**         | {{ optionlink('extensions.image') }} |
+| **Value**       | string |
+| **Example**     | `perconalab/percona-postgresql-operator:{{ release }}-ext-installer` |
```

Comment:
this example is wrong, users need to set it to operator image i.e `percona/percona-postgresql-operator:2.3.0`